src/Cake/Scripting/Roslyn/Stable/RoslynScriptSessionFactory.cs

```diff
@@ -10,15 +10,15 @@ namespace Cake.Scripting.Roslyn.Stable
 {
     using Core.IO;
```

Comment:
@devlead yep, all of the above have been like this, it just got duplicated when i splitted it into and abstract class + derived classes. If it's possible to move it outside the namespace, then I'll move them.